optimize.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/sh
 
-for file in contracts/*; do
-    name=$( echo "${file#contracts/}" | tr '-' '_' )
-    wasm_file="target/wasm32-unknown-unknown/release/${name}.wasm"
-    if [ -f "$wasm_file" ]; then
+release_folder="target/wasm32-unknown-unknown/release"
+prefix="stellar_"
+
```

Comment:
nit: Perhaps gracefully warn if stellar not installed
```suggestion
if ! command -v stellar >/dev/null 2>&1; then
    echo "Error: 'stellar' command not found" >&2
    exit 1
fi
```

Reply:
nit: if release_folder DNE
```suggestion
if [ ! -d "$release_folder" ]; then
    echo "Error: Release folder not found: $release_folder" >&2
    exit 1
fi
```